hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/shims/Hive2Shim.java

```diff
@@ -53,4 +59,17 @@ public int getDays(Object dateWritable) {
   public long getMills(Object timestamp) {
     return ((Timestamp) timestamp).getTime();
   }
+
+  @Override
+  public PartitionDesc getPartitionDesc(Map<Path, PartitionDesc> pathToPartitionInfo, Path dir,
+      Map<Map<Path, PartitionDesc>, Map<Path, PartitionDesc>> cacheMap) throws IOException {
+
+    try {
+      Class<?> hiveUtilsClass = Class.forName("org.apache.hudi.hadoop.utils.HudiHiveFileFormatUtils");
```

Comment:
Do we need reflection here?

Reply:
@xicm  has been modified to call the method directly